Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -388,7 +393,9 @@ namespace Multiplayer
         stats.m_clientConnectionCount = 0;
 
         // Send out the game state update to all connections
-        {
+        {            
+            AZ_PROFILE_SCOPE(MULTIPLAYER, "SendOutGameStateUpdateToAllConnections");
```

Comment:
For submarkers, would also add context so folks don't have to look in the code, it should be clear from marker what this represents IMHO.